.github/workflows/ci.yml

```diff
@@ -27,7 +27,7 @@ jobs:
         python -m pip install --upgrade pip
         pip install https://github.com/pyro-ppl/funsor/archive/master.zip
         pip install jaxlib
-        pip install jax
+        pip install "jax<0.2"
```

Comment:
It will be best to pin this to `0.1.77` because I am pretty sure our current dev branch will be incompatible with older versions of JAX.

Reply:
Yeah, I just made the change.